app/views/bulletins/_form.html.erb

```diff
@@ -28,7 +24,7 @@
 
   <div class="field">
     <%= form.label :priority %>
-    <%= form.text_field :priority %>
+    <%= form.radio_button :priority, "Priority" %>
```

Comment:
If this is just for true/false values I think usually checkboxes are preferred over radios since radios can't be unchecked if they are hit by accident.